pipeline/pipeline/config.py

```diff
@@ -9,7 +9,8 @@
 dropbox_client = DropboxClient(DROPBOX_TOKEN)
 
 GSPREAD_CLIENT = None
-DEFAULT_WORKSHEET_URL = "https://docs.google.com/spreadsheets/d/1HeTZKEXtSYFDNKmVEcRmF573k2ZraDb6DzgCOSXI0f0/edit#gid=0"
+#! DEFAULT_WORKSHEET_URL = "https://docs.google.com/spreadsheets/d/1HeTZKEXtSYFDNKmVEcRmF573k2ZraDb6DzgCOSXI0f0/edit#gid=0"
```

Comment:
This one needs to be enabled right to have it update Phase 1 instead of your tests on Phase 2 sheet?